### PR TITLE
Add statcounter to statistics

### PR DESCRIPTION
**Pull request by davidgs**
### Issue
No issue created
### Description
[statcounter](https://statcounter.com) is a popular web stats site. Integrating it into toha was (previously) a pain, so this PR makes it super simple.

It follows the same configuration rules as the other web analytics options. 

### Test Evidence

statcounter-2
statcounter-1

The configuration parameters are picked up and put into the stat counter code in the header, as expected.

### CODE REVIEW

**Comment by hossainemruz** [comment]
Thank you @davidgs! Look like the PR contains changes from you other PR. Can you please fix it?

Also, can you please guides here: https://github.com/hugo-toha/guides/blob/main/content/posts/analytics/index.md

**Comment by davidgs** [comment]
Should be all updated. Added a PR to the guide, too

**Comment by hossainemruz** [comment]
Can you please resolve the conflict?

**Comment by davidgs** [comment]
resolved.

File: layouts/partials/misc/canoncial.html
Comment (hossainemruz) [inline layouts/partials/misc/canoncial.html:1]: This file should not be part of this PR right?

Comment (hossainemruz) [reply layouts/partials/misc/canoncial.html:1]: Just delete it and commit the change.

**Review by hossainemruz — APPROVED** [review]
Thank you @davidgs! Looks good now.
